pandas/tests/strings/test_find_replace.py

```diff
@@ -220,14 +220,18 @@ def test_contains_nan(any_string_dtype):
     expected = Series([True, True, True], dtype=expected_dtype)
     tm.assert_series_equal(result, expected)
 
-    result = s.str.contains("foo", na="foo")
-    if any_string_dtype == "object":
-        expected = Series(["foo", "foo", "foo"], dtype=np.object_)
-    elif any_string_dtype == "string[pyarrow_numpy]":
-        expected = Series([True, True, True], dtype=np.bool_)
+    if any_string_dtype == "string[python_numpy]":
+        with pytest.raises(TypeError):
```

Comment:
Why does this variant raise a TypeError?

Reply:
It does not anymore, so updated the test